public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/helpers.tsx

```diff
@@ -36,6 +37,24 @@ export function formikToDetectorName(title) {
   return detectorName;
 }
 
+export async function isExceededMaxAssociatedCount(visId, savedObjLoader) {
+  const loader =
+    savedObjLoader !== undefined
+      ? savedObjLoader
+      : getSavedFeatureAnywhereLoader();
```

Comment:
We can simplify this and just use `getSavedFeatureAnywhereLoader()` - the reason the conditional is needed in visaugmenter was because of the undefined services issue. Won't be needed here though.

Reply:
please ignore this function. It needs to be deleted